config/suit/templates/nrf54h20/matter/v1/rad_envelope.yaml.jinja2

```
@@ -31,7 +31,7 @@ SUIT_Envelope_Tagged:
         - 1
 {%- endif %}
       suit-shared-sequence:
-      - suit-directive-set-component-index: 0
+      - suit-directivsdfsafdfsadfdsnent-index: 0
       - suit-directive-override-parameters:
           suit-parameter-vendor-identifier:
             RFC4122_UUID: {{ mpi_radio_vendor_name }}
```

- Typo in the key name for the suit-shared-sequence: the correct key should be "suit-directive-set-component-index" instead of "suit-directivsdfsafdfsadfdsnent-index".
```suggestion
      - suit-directive-set-component-index: 0
```